stable/jenkins/templates/_helpers.tpl

```diff
@@ -72,19 +72,21 @@ Returns configuration as code default config
 */}}
 {{- define "jenkins.casc.defaults" -}}
 jenkins:
-  disableRememberMe: false
+  disableRememberMe: {{ .Values.master.disableRememberMe }}
```

Comment:
please also add `master.disableRememberMe` to the XML configuration [here](https://github.com/helm/charts/blob/b1892ec50bcd66e917c0f3991517e0fd3dc68b69/stable/jenkins/templates/config.yaml#L31)

Reply:
Done